debian/bitlbee-common.preinst

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+
+set -e
+
+NAME=bitlbee
+PIDFILE=/var/run/$NAME.pid
+
+# git revision 7949d5a introduced dh_systemd.
+DH_SYSTEMD_REV="3.4.2+20160612+master+41-g7949d5a-git"
```

Comment:
Argh, nevermind. The variable name confused me. You're comparing the bitlbee versions and not the dh_systemd versions. Ignore me.